R/method-filterFeatures.R

```diff
@@ -0,0 +1,622 @@
+#' @title Filtering of features based on conventional quality assessment
+#'
+#' @description
+#'
+#' When dealing with metabolomics results, it is often necessary to filter
+#' features based on certain criteria. These criteria are typically derived
+#' from statistical formulas applied to full rows of data, where each row
+#' represents a feature and its abundance of signal in each samples.
+#' The `filterFeatures` function filters features based on these conventional
+#' quality assessment criteria. Multiple types of filtering are implemented and
+#' can be defined by the `filter` argument.
+#'
+#' Supported `filter` arguments are:
+#'
+#' - [`RsdFilter`]: Calculates the relative standard deviation
+#'  (i.e. coefficient of variation) in abundance for each feature in QC
+#'  (Quality Control) samples and filters them in the input object according to
+#'  a provided threshold.
+#'
+#' - [`DratioFilter`]: Computes the D-ratio or *dispersion ratio*, defined as
+#'  the standard deviation in abundance for QC samples divided by the standard
+#'  deviation for biological test samples, for each feature and filters them
+#'  according to a provided threshold
```

Comment:
sorry, now I'm really becoming picky - but could you please add a . at the end of the sentence?